draft-kalos-bbs-blind-signatures.md

```diff
@@ -364,44 +366,43 @@ Deserialization:
 
 Procedure:
 
-1. message_scalars = ()
-2. if secret_prover_blind != 0, message_scalars.append(
+1. generators = BBS.create_generators(L + 1, api_id)
+2. blind_generators = BBS.create_generators(M + 1, "BLIND_" || api_id)
+
+3. message_scalars = BBS.messages_to_scalars(messages, api_id)
+
+4. blind_message_scalars = ()
+5. if secret_prover_blind != 0, blind_message_scalars.append(
```

Comment:
This addition has to be done in the scalar field. Don't know if that is obvious from the context but I forgot to do this in my implementation.